apkid/rules/elf/packers.yara

```diff
@@ -852,6 +852,26 @@ rule appguard_elf : packer
     is_elf and any of them
 }
 
+rule appguard_elf_b : packer
+{
+  meta:
+    description = "AppGuard"
+    url         = "http://appguard.nprotect.com/en/index.html"
+    sample      = "a6e9c876be2b8b936ab9bfe2699811524b8ad3c11305099b34194bb8aad79f1e"
+    sample2     = "23cd2af10d46459065ea65b2d40fb706fd4847a1f8ef195cbebf1c6d8d54a48a"
+    author      = "Moolakarapaiyan"
+
+  strings:
+    $a = { 6C 69 62 63 6F 6D 70 61 74 69 62 6C 65 2E 73 6F 00 }  // libcomptabile.so
+    $b = { 00 23 4C 63 6F 6D 2F 69 6E 63 61 2F 73 65 63 75
```

Comment:
Just noticed this, are we sure the hex 23 (ASCII #) is always there ?

This is an elf rule

Reply:
No, I believe yes, since ELF doesn't have a rule that follows by length + string, to avoid missing future detections.